src/components/Stays/StaysRating.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import { HSpace } from "@components/shared/HSpace";
+import { Icon } from "@components/shared/Icon";
+
+export interface StaysRatingProps {
+  rating: number | null;
+  small?: boolean;
+}
+
+export const StaysRating: React.FC<StaysRatingProps> = ({ rating, small }) => {
+  if (!rating) {
+    return null;
```

Comment:
I would throw instead of returning on the library. that way those integrating know exactly why there's an issue. If there need to be a guard to check if this is present or not, it should happen outside of the component imo. Is that fair?

 ```suggestion
  if (!rating) {
    throw new Error('Attempted to render StaysRating without a rating present')
```